{"name": "test_six[6-9-13]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 15 == 13\n +  where 15 = add(6, 9)", "trace": "self = <Work.test_work2.TestB object at 0x104e76f98>, module = None\nclassB_start = None, a = 6, b = 9, expected = 13\n\n    @pytest.mark.parametrize(\"a,b,expected\", json.load(open(\"calc.json\")))\n    def test_six(self, module, classB_start, a, b, expected):\n        # a = json.load(open(\"calc.json\"))\n        # print(a[\"login\"])\n        logging.info(\"This is test_six!!!\")\n>       assert add(a, b) == expected\nE       assert 15 == 13\nE        +  where 15 = add(6, 9)\n\ntest_work2.py:45: AssertionError"}, "attachments": [{"name": "log", "source": "6dfa906e-c44f-42a3-9abe-fb933e368e7b-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "af49d3c4-cccf-46c2-a336-5fae0f49e2fa-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "6"}, {"name": "b", "value": "9"}, {"name": "expected", "value": "13"}], "start": 1572059932563, "stop": 1572059932564, "uuid": "e5ea96cc-62a9-4fe6-b5b9-bb401518866f", "historyId": "ee7ec1a16f5c08ad1f28f68816b4f5b5", "testCaseId": "4f6de44d5dec1d8b69075d2b48ad4434", "fullName": "test_work2.TestB#test_six", "labels": [{"name": "suite", "value": "test_work2"}, {"name": "subSuite", "value": "TestB"}, {"name": "host", "value": "ParsondeMacBook-Pro.local"}, {"name": "thread", "value": "7295-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_work2"}]}